### PR TITLE
Update type of custom element constructor

### LINKED ISSUE #20875: Update type of custom element constructor
https://github.com/whatwg/html/pull/3703

### PR DESCRIPTION
**Pull request by tigercosmos**
---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #20875 (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/20879)

### CODE REVIEW

File: components/script/dom/webidls/CustomElementRegistry.webidl [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
   [Throws, CEReactions]
-  void define(DOMString name, Function constructor_, optional ElementDefinitionOptions options);
+  void define(DOMString name, CustomElementConstructor constructor_, optional ElementDefinitionOptions options);
 
   any get(DOMString name);
 
   Promise<void> whenDefined(DOMString name);
 };
 
+callback CustomElementConstructor = any(any... arguments);
```
Comment (jdm) [inline components/script/dom/webidls/CustomElementRegistry.webidl]: Why does this include `any... arguments`?

Comment (jdm) [reply components/script/dom/webidls/CustomElementRegistry.webidl]: `any()` means that it is a function that can return any value but accepts no arguments.

**Comment by tigercosmos** [comment]
I have let any() to be empty.

**Pull request merged**